spi/src/main/java/org/opensearch/jobscheduler/spi/LockModel.java

```diff
@@ -88,6 +94,18 @@ public LockModel(String jobIndexName, String jobId, Instant lockTime,
         this.primaryTerm = primaryTerm;
     }
 
+    public LockModel(String jobIndexName, String jobId, String lockId, Instant lockTime,
```

Comment:
why do we want both jobId and lockId?

Reply:
That's a good point. I think that the only reason to include the job ID would be if we wanted to delete the leftover locks after a job is deleted. If we instead moved to a periodic job deleting stale locks, then I think there is no point in having both. I'll make the change.